benchmarking/benchmark_framework_module.rb

```diff
@@ -0,0 +1,22 @@
+module BenchmarkFrameworkModule
+  class BenchmarkFramework
+    @@blocks = {}
+    class << self
```

Comment:
This is an interesting pattern that I have not seen much of, it's a cool shortcut.